bitcoin/src/blockdata/transaction.rs

```diff
@@ -1010,7 +1010,7 @@ impl InputWeightPrediction {
     /// The function panics in const context and debug builds if `bytes_to_grind` is higher than 62.
     ///
     /// [signature grinding]: https://bitcoin.stackexchange.com/questions/111660/what-is-signature-grinding
-    pub const fn ground_p2wpkh(bytes_to_grind: usize) -> Self {
+    pub const fn ground_p2wpkh(bytes_to_grind: u32) -> Self {
```

Comment:
Notice that this is at most 62, thus it should've been `u8` from the start, not `usize`.